.dockerignore

```diff
@@ -0,0 +1,15 @@
+# Ignore Dockerfile
+*.Dockerfile
+Dockerfile
+
+# Ignore docker-compose files
+docker-compose.yml
+docker-compose.override.yml
+
+# Ignore .git and .cache directories
```

Comment:
can we ignore .git? seems to reduce the build context by ~400mg (if not, remove comment)